src/lib/react.js

```diff
@@ -14,7 +14,22 @@ options.vnode = vnode => {
 	if (old) old(vnode);
 };
 
-const Children = { only: c => c[0], count: c => c.length };
+const childrenForEach = (children, cb) =>
+	Array.isArray(children) && children.forEach((VNode) => {
+		// this is gross, send help
```

Comment:
Ah yeah we might want to just have this lib define `VNode.prototype.props` as a getter.